alg/gdalwarpoperation.cpp

```diff
@@ -2970,7 +2985,7 @@ CPLErr GDALWarpOperation::ComputeSourceWindow(
 #if DEBUG_VERBOSE
     CPLDebug(
         "WARP",
-        "dst=(%d,%d,%d,%d) raw src=(minx=%.8g,miny=%.8g,maxx=%.8g,maxy=%.8g)",
+        "dst=(%d,%d,%d,%d) raw src=(minx=%.18g,miny=%.18g,maxx=%.18g,maxy=%.18g)",
```

Comment:
😍

Reply:
don't get too excited by that: that's just a debug trace not enabled in normal builds :-)

Reply:
😂